pkg/api/v1alpha1/users.go

```diff
@@ -217,6 +232,19 @@ func (r *Router) createUser(c *gin.Context) {
 		user.Status = null.StringFrom(UserStatusPending)
 	}
 
+	wg := &sync.WaitGroup{}
+	updateNotificationPreferencesStatusCode := http.StatusOK
+	updateNotificationPreferencesError := error(nil)
+
+	wg.Add(1)
+
+	updateNotificationPreferences := func() {
+		defer wg.Done()
+
+		_, updateNotificationPreferencesStatusCode, updateNotificationPreferencesError = handleUpdateNotificationPreferencesRequests(c, r.DB, user, r.EventBus, req.NotificationPreferences)
+	}
+	go updateNotificationPreferences()
```

Comment:
does this really need to be async? if these are slow, would it be better to make a separate call?